recipes/isitgr/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "isitgr" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mishakb/ISiTGR/archive/{{ version }}.tar.gz
+  sha256: ebf53b46ce3682a0154f3e713e8d320f5f7588fe1cb7a6fe5c9db8d1cad78b59
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('fortran') }}  # [not win]
+#    - m2w64-toolchain  # [win]
+    - git  # [not win]
+#    - m2-git  # [win]
+    - make  # [not win]
+#    - m2-make  # [win]
+    - python
+    - setuptools
+    - pip
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy
+    - six  # TODO: remove for next update
+    - mock
+    - setuptools  # for an import of pkg_resources
+    - sympy >=1.0
+    - scipy >=1.0
+    - {{ compiler('fortran') }}  # [not win]
+
+test:
+  imports:
+    - isitgr
+  files:
+    - test_isitgr.py
+    - run_test.sh  # [win]
+
+about:
+  home: https://github.com/mishakb/ISiTGR
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Does this vendor `camb`? If so you need to include it's licence in the conda package and modify the `license` field above:
```suggestion
  license_file:
    - LICENSE
    - camb/LICENCE.txt
```

---

The licence in SPDX then becomes `MIT AND GPL-3.0-only` (or whatever is appropriate). You can delete the `licence_family` field.